en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -472,4 +473,76 @@ To create a For Each Loop:
+.. If you create a new flow:
+... Select the connector or system to start your flow.
+... Select and configure a trigger. 
+... Select and configure an action.
+... Directly before the step to which you want to add an error handler, click the plus button. 
+... In the *What Should Start This Flow?* window, click *Error Handler (Beta)*.
+.. If you add a step to an existing flow:
+... Open your flow.
+... Directly before the step to which you want to add an error handler, click the plus button. 
+... In the *Next Step* window, click *Error Handler (Beta)*. 
+
+[[watch-a-step]]
+=== Watch a Step 
+
+You can watch specific steps for connection or logic errors. If an error occurs, the flow stops normal processing and begins processing the steps in the error handler. You can add one or more steps to watch in this section.
+
+To watch a step: 
+
+. <<access-error-handling,Access the error handler in your flow>>.
+. In the *Error Handler* card, in the *Watch* section, click the plus sign and select the action to watch or drag the step to watch into the *Watch* section. +
+The selected action appears in the *Watch* section. If an error occurs, the flow moves to the actions selected in the *On Error* section.
+. To add additional actions to watch, repeat the previous step.
+
+=== Configure Composer to Handle Errors 
+
+You can configure Composer to handle errors encountered in the *Watch* section. If an error occurs in a watched action, the *On Error* section begins processing the flow based on your configuration. You can add one or more steps to run in the *On Error* section. 
+
+==== Error Handling Behavior
+
+* By default, the *On Error* section allows for one condition to be met; however, you can add as many conditions as you want in this section. 
+* When adding a new condition, the *On Error* condition field value is set to `None` by default. Leaving this value set to `None` returns any errors that may occur in the *Watch* section and processes all steps inside this *On Error* section. 
+* When a condition is met in the *On Error* section, the error handler processes the conditions in sequential order and stops once the first condition is met. After the first condition is met, any subsequent steps are skipped. If you want the process to continue, you can add an additional *On Error* section branch inside your existing *On Error* section.
+* When xref:ms_composer_test_flow.adoc[testing your flow], any errors caught by the error handler appear in the right pane of the action card. Successful steps are marked with a green check and failed steps are marked with a red X. 
+* When an error occurs in a watched action, the error information is saved as a data pill that can then be used for subsequent notification or record-updating actions. For example, if an error occurs, you can specify an action to send an email to your team containing the error information from the data pill.
+
+==== Steps 
```

Comment:
I forgot to mention that we no longer use this in the content type templates. Suggest changing to "Add Steps to the On Error Section" (or something else descriptive).